<?xml version="1.0"?>
<launch>

  <arg name="depth_image_topic"/>
  <arg name="color_image_topic"/>
  <arg name="camera_info_topic"/>
  <arg name="cache_count" default="10"/>
  <arg name="slop" default="0.01"/>

  <node pkg="industrial_reconstruction" exec="industrial_reconstruction_hybrid" name="industrial_reconstruction_hybrid" output="screen">
    <param name="depth_image_topic" value="$(var depth_image_topic)"/>
    <param name="color_image_topic" value="$(var color_image_topic)"/>
    <param name="camera_info_topic" value="$(var camera_info_topic)"/>
    <param name="cache_count" value="$(var cache_count)"/>
    <param name="slop" value="$(var slop)"/>
  </node>

</launch>
